certified-connectors/iObeya/apiDefinition.swagger.json

```diff
@@ -0,0 +1,598 @@
+          {
+            "name": "type",
+            "type": "string",
+            "required": true,
+            "in": "query"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ok",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/qcd/indicators-values": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default"
+          }
+        },
+        "summary": "Update QCD Indicators",
+        "description": "Bulk Update of QCD Indicators Values",
+        "operationId": "UpdateQCDIndicatorsValue",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "required": [
+                "boardId",
+                "letters"
+              ],
+              "properties": {
+                "boardId": {
+                  "type": "string",
+                  "description": "ID of the board where place the card",
+                  "title": "Board ID",
+                  "x-ms-visibility": "important",
+                  "x-ms-dynamic-values": {
+                    "operationId": "ListBoards",
+                    "value-path": "id",
+                    "value-collection": "data",
+                    "value-title": "name"
+                  }
+                },
+                "letters": {
+                  "type": "array",
+                  "items": {
+                    "type": "object"
+                  },
+                  "description": "Indicators",
+                  "title": "Indicators",
+                  "x-ms-visibility": "important"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/qcd/compute-indicator": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default"
+          }
+        },
+        "summary": "Compute QCD Indicator",
+        "operationId": "ComputeQCDIndicator",
+        "description": "Compute a QCD Indicator object",
+        "parameters": [
+          {
+            "name": "letterName",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Letter Name",
+            "x-ms-summary": "Letter Name"
+          },
+          {
+            "name": "indicatorName",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Indicator Name",
+            "x-ms-summary": "Indicator Name"
+          },
+          {
+            "name": "wedgeValue",
+            "in": "query",
+            "required": true,
+            "type": "number",
+            "description": "Indicator Value",
+            "x-ms-summary": "Indicator Value"
```

Comment:
@smiquerolle summaries and descriptions should not look like a copy and paste. Please refer to English strings [guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).